cli/cli.go

```diff
@@ -37,11 +37,14 @@ const (
 	idTokenOption         = "idtoken"
 	cliClientID           = "github.com-vmware-priam"
 	cliClientSecret       = "not-a-secret"
+	defaultAwsCredFile    = ".aws/credentials"
+	defaultAwsProfile     = "default"
```

Comment:
I dunno. I would expect the default profile to be used if a profile is not specified. It seems odd to me to have to specify a profile of "default". Perhaps we should ask some users who are planning to use this feature?